master/buildbot/www/avatar.py

```diff
@@ -72,12 +82,15 @@ def render_GET(self, request):
     def renderAvatar(self, request):
         email = request.args.get(b"email", [b""])[0]
         size = request.args.get(b"size", 32)
+        username = request.args.get(b"username")
```

Comment:
maybe we should use the same as line 83...

```suggestion
        username = request.args.get(b"username", [None])[0]
```